git_theta/updates/base.py

```diff
@@ -26,7 +26,8 @@ class Update(metaclass=ABCMeta):
 
     name: str = NotImplemented  # The name used to lookup the plug-in.
 
-    def __init__(self, serializer: params.Serializer, *args, **kwargs):
+    def __init__(self, path: str, serializer: params.Serializer, *args, **kwargs):
```

Comment:
The base class never needs the path right? Can we move this to IncrementalUpdate __init__ like the `update_data` parameter?